packages/mui-system/src/cssVars/cssVarsParser.ts

```diff
@@ -77,6 +77,10 @@ const getCssValue = (keys: string[], value: string | number) => {
       // CSS property that are unitless
       return value;
     }
+    if (keys[keys.length - 1].indexOf('Channel') !== -1) {
+      // the channels are unitless too
+      return value;
+    }
```

Comment:
I would change this logic to check the value instead. There are only 2 formats of the channel.

```js
0 0 0
0,0,0
```
even though the first one is the latest but both of them are supported in the modern browsers.

Reply:
Actually, we should return `${value}px` only if the value is a valid number, no? In all other cases we should return the value itself. I will double check this to confirm

Reply:
Agree, adding tests is nice.

Reply:
Done